Description: Disabled hginstallscripts @LIBDIR@ replacement in setup.py.
 setup.py replaces @LIBDIR@ in the hg script, with a path that differs between
 Python verisons.
 libdir in hg doesn't need to be set if mercurial is available in the public
 namespace, as it is in Debian.
 hg doesn't alter sys.paths if this replacement hasn't happened.
Author: Stefano Rivera <stefanor@debian.org>
Bug-Debian: http://bugs.debian.org/620087
Bug-Ubuntu: https://bugs.launchpad.net/bugs/745250
Forwarded: not-needed

Index: mercurial-4.5/setup.py
===================================================================
--- mercurial-4.5.orig/setup.py	2018-02-11 07:36:52.182893969 +0200
+++ mercurial-4.5/setup.py	2018-02-11 07:36:52.182893969 +0200
@@ -778,7 +778,6 @@
             'build_hgextindex': buildhgextindex,
             'install': hginstall,
             'install_lib': hginstalllib,
-            'install_scripts': hginstallscripts,
             'build_hgexe': buildhgexe,
             }
 
